openapi.yml

```diff
@@ -5569,6 +5521,89 @@ components:
                 pagination:
                     $ref: '#/components/schemas/Pagination'
                     description: ''
+        GuestAlertCreateParams:
+            description: ''
+            required:
+                - message
+                - channels
+            type: object
+            properties:
+                message:
+                    description: Plain text of the alert to be sent
+                    type: string
+                channels:
+                    description: 'Specify the broadcast channel, one of ''SMS'' and/or ''EMAIL'''
+                    type: array
+                    items:
+                        type: string
+                signin_selectors:
+                    $ref: '#/components/schemas/GuestAlertSigninSelectors'
+                    description: ''
+            example:
+                message: some text
+                channels:
+                    - SMS
+                    - EMAIL
+                signin_selectors:
+                    signin_ids:
+                        - 19
+                        - 64
+                    is_signed_out: false
+        GuestAlertSigninSelectors:
+            description: ''
+            type: object
+            properties:
+                signin_ids:
+                    description: ''
+                    type: array
+                    items:
+                        type: integer
+                is_signed_out:
+                    description: ''
+                    type: boolean
+        GuestAlertDetail:
+            description: ''
+            required:
+                - delivery_status
+                - channel
+                - pending_count
+                - failed_count
+                - sent_count
+                - guest_alert_uuid
```

Comment:
no need to say `guest_alert` here, it's already known as a `GuestAlert` model
`uuid` is better